File: typescript-angular/API/Client/CloudFile.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface CloudFile {

        "id"?: string;

        "type"?: string;

        "aCL"?: ACL;

        "name"?: string;

        "size"?: number;

        "url"?: string;

        "expires"?: Date;

        "contentType"?: string;
    }

}

File: scala/src/main/scala/io/swagger/client/model/CloudFile.scala
package io.swagger.client.model

import org.joda.time.DateTime



case class CloudFile (
  id: String,
  _type: String,
  ACL: ACL,
  name: String,
  size: Float,
  url: String,
  expires: DateTime,
  _contentType: String)
  
